test/PlatformColor.js

```diff
@@ -0,0 +1,27 @@
+import { PlatformColor, DynamicColorIOS } from 'react-native';
+
+// Invalid
+// eslint-disable-next-line @react-native/platform-colors
+PlatformColor(); // required at least one argument
+// eslint-disable-next-line @react-native/platform-colors
+PlatformColor([], {}); // arguments must be string literals
+// eslint-disable-next-line @react-native/platform-colors
+DynamicColorIOS('red'); // required object as first argument
```

Comment:
tbh looks like TS should take care of these

Reply:
of course TS will catch it early 
but it still useful for JS projects